libqtile/widget/mpdwidget.py

```diff
@@ -263,17 +244,19 @@ def _get_status(self):
             if self.status['state'] != 'stop':
                 text = self.do_format(self.fmt_playing)
 
-                if (self.do_color_progress and
-                        self.status and
-                        self.status.get('time')):
-                    elapsed, total = self.status['time'].split(':')
-                    percent = float(elapsed) / float(total)
-                    progress = int(percent * len(text))
-                    playing = '<span color="%s">%s</span>%s' % (
-                        utils.hex(self.foreground_progress),
-                        pangocffi.markup_escape_text(text[:progress]),
-                        pangocffi.markup_escape_text(text[progress:])
-                    )
+                if self.do_color_progress and self.status.get('time'):
+                    try:
+                        elapsed, total = self.status['time'].split(':')
+                        percent = float(elapsed) / float(total)
+                        progress = int(percent * len(text))
+                    except Exception:
```

Comment:
At least division by 0, and maybe also if `time` is not correctly formated and doesn't contain `:` - which will raise a `ValueError` but I don't know if it can occur. I agree with not having a bare excpet clause... What do you suggest to do instead? Catching only those two exceptions?

Thanks for your review by the way!